### ISSUE TITLE
The SR result seems dirty.

### ISSUE THREAD

**Issue by feature-space-move** [Guaishou74851/AdcSR#26]
The left image is origin image, and the right is SR reslult.
<img width="3852" height="2446" alt="Image" src="https://github.com/user-attachments/assets/e8f46ab5-9bb7-467e-ab37-9c46c41cd821" />

**Comment by tvelovraf** [comment]
This is the result of adversarial loss use during training, as I understand.
You can try to finetune few epochs without it, and these "stains" should go away
